packages/kotti-input-number/src/InputNumber.vue

```diff
@@ -1,30 +1,24 @@
 <template>
 	<div :class="formGroupStyle">
-		<div
-			:class="decreaseButtonStyle"
-			@click="setCurrentValue(currentValue - step)"
-		>
+		<div :class="decreaseButtonStyle" @click="decrementValue">
 			<i class="yoco">minus</i>
 		</div>
 		<input
 			ref="inputNumber"
-			:class="inputStyle"
 			type="number"
+			:min="min"
```

Comment:
adding the type and min/max would introduce two buttons if i'm not mistaken

Reply:
ha so i have return it :).

it didn't - i thought this was the bug i was seeing - but apparently not. The bug was that for some reason computing the decrease limit was breaking. I tried more elegant solutions first. I'll tell you in our meet because its long.